app/src/main/java/org/greenstand/android/TreeTracker/permissions/PermissionViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package org.greenstand.android.TreeTracker.permission
+
+import android.location.LocationManager
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+data class PermissionItemsState(
+    val openAppSettings: Boolean = true,
+    val isLocationEnabled: Boolean? = null
```

Comment:
I think we can set this to false by default.

Reply:
The parameter "openAppSettings" is no longer being used as a state object so I deleted it in a new commit.

Reply:
I was refering to the isLocationEnabled. It's set to null, but can be set to false by default. This will allow you to avoid dealing with nullability. 
It's why you have 
if(state.isLocationEnabled == false)
instead of
if(state.isLocationEnabled)